PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/functionapp/config/FunctionAppRunState.kt

```diff
@@ -125,6 +126,30 @@ class FunctionAppRunState(project: Project, private val myModel: FunctionAppSett
         return FunctionAppDeployResult(app, database)
     }
 
+    private fun tryConfigureAzureFunctionRuntimeStack(app: WebAppBase, subscriptionId: String, processHandler: RunProcessHandler) {
+        if (app !is FunctionApp) return
+
+        val functionRuntimeStack = myModel.functionAppModel.functionRuntimeStack
```

Comment:
This method updates the runtime settings in Azure.

Reply:
This logic raise one more request to the server. Do we have an option to configure an app on create request using the same runtime configuration logic? Or is this API available after app creation only?

Reply:
Correct. If you update your project TFM or change from isolated to default worker, we have to make sure it all works.